### ISSUE TITLE
Incorrect Cape Cod reserve when incurred loss = 0

### ISSUE THREAD

**Issue by cdietrich215** [casact/chainladder-python#321]
I noticed a minor issue when reserving using the Cape Cod. When incurred or paid loss is $0 for an accident period, the Cape Cod method will predict $0 of Ultimate Loss, when theoretically there should still be a reserve > $0. This may matter for some smaller insurers.

The BF and Benktander methods in this package both correctly set ultimate losses to be (a priori) * (sample weight) * (1 - 1/CDF) in these situations. I believe that the Cape Cod method should do the same. This method currently sets IBNR to 0 (or null).

**Comment by jbogaardt** [comment]
Thanks. I didn't know about this error. 

Reproducible example:
```python
import chainladder as cl

triangle = cl.load_sample('raa')
premium = triangle.latest_diagonal * 0 + 50000
triangle.loc[:,:,'1987',48] = 0
cl.CapeCod().fit(triangle, sample_weight=premium).ultimate_
```

**Comment by kennethshsu** [comment]
Can close the ticket once #368 is merged.

**Comment by jbogaardt** [comment]
Reopening until release of 0.8.14
